SU-HSDO/suhumsci
# fix(STN-848): making raised hb-cards uniform in width as well when a

# READY FOR REVIEW

## Summary
Add `flex-grow: 1;` to `.hb-raised-cards .hb-card`

## Need Review By (Date)
ASAP

## Urgency
Medium

## Steps to Test
1. Replicate [this](https://southasia.stanford.edu/peoplefaculty/core-faculty) page locally
2. Switch over to Browserstack and load your local page with Firefox 83 to confirm that all of your cards are indeed filling up the available space equally.
3. Switch to Chrome 85 and repeat check from step 2.
4. I would also think of any other ways existing implementations could be effected by adding `flex-grow`. I was unable to locate any adverse consequences, but your mileage may vary.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
